visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html

```diff
@@ -1,4 +1,4 @@
-<md-input-container>
```

Comment:
I am not certain what `Edge metric` stands for. Do you know of any other description? This is not a blocker though.

Reply:
We have 5 metric choosers. Distribution, Area, Height, Color and the Edge metric.